whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -2976,7 +2969,16 @@ class PostgreSQLComponent {
             }
         }
     }
-    
+
+    private static void assertNoLinksToDeleted(Set<Link> links) {
+        links.each {link ->
+            // Except sameAs since it's sometimes necessary to add sameAs pointing at already deleted records
```

Comment:
```suggestion
            // sameAs is allowed because when merging two entities, the id of the deleted entity is added to sameAs of the remaining entity
```